packages/core/src/components/forms/_label.scss

```diff
@@ -8,19 +8,22 @@
 /*
 Labels
 
-Add labels to inputs to enhance the usability of your forms.
+Labels enhance the usability of your forms.
 
-Putting the `<input>` element _inside_ a `<label>` element also increases the area where the user
-can click to activate the control. Notice how in the example below, clicking a `<label>` below
-focuses its `<input>`.
+- Add extra information to the label with `span.pt-label-description`.
+
+- Putting the `<input>` element _inside_ a `<label>` element increases the area where the user
+can click to activate the control. Notice how in the examples below, clicking a `<label>` focuses its `<input>`.
```

Comment:
Frankly, my dream scenario is to find a way to support arbitrary form elements (`pt-input`, `pt-control-group`, `pt-button-group`, etc.) inside of a `label`. From a user perspective, I'd expect at least that common pattern to work, and I'd be thrown off by the current behavior of `pt-control-group` in that context.

Barring that, the best approach in my mind is to keep `label` around as a "leaf" element (one that can't have children) and require use of `pt-form-group` for the `for`-with-`id` use case.

So I guess I'm arguing for No. 1 above.